### DISCUSSION TITLE
Proposal to merge meta and utils repositories into a single, unified meta repository

### DISCUSSION THREAD

**Discussion by IgorTodorovskiIBM** [zopencommunity/meta#108] [General]
We are proposing to consolidate the z/OS Open Tools [utils](https://github.com/ZOSOpenTools/utils) and [meta](https://github.com/ZOSOpenTools/meta) repositories into a single, unified meta repository. This is to streamline communication and to bring all discussions/tooling to a central location.
A new branch has been created for this merger and can be found at https://github.com/ZOSOpenTools/meta/tree/merged. Additionally, we have developed a tool to seamlessly migrate any issues from the utils repository to the meta repository.
Please let us know if you have any concerns or have feedback. If all is clear, we will proceed by replacing the main branch with the merged branch this Friday evening.

**Comment by MikeFultonDev** [comment]
Can we mark this 'closed' now @IgorTodorovskiIBM ?

**Comment by IgorTodorovskiIBM** [comment]
Yes!

Utils has been merged with Meta.  We have deprecated the utils repo for now and we will likely delete it at a later point.

Also, it looks like 'closing a discussion' is a yet to be implemented github feature: https://github.com/orgs/community/discussions/3097 .
